### PR TITLE
Clean up messages to make them proper English sentences

### PR DESCRIPTION
**Pull request by retroflexer**

### CODE REVIEW

File: pkg/operator/etcdmemberscontroller/etcdmemberscontroller.go [hunk 1]
```diff
@@ -140,7 +140,18 @@ func (c *EtcdMembersController) reportEtcdMembers() error {
 		}
 	}
 
-	if len(availableMembers) > len(etcdMembers)/2 {
+	if len(availableMembers) == len(etcdMembers) {
```
Comment (alaypatel07) [inline pkg/operator/etcdmemberscontroller/etcdmemberscontroller.go]: what is the status output when 1 member (or minority members) is unhealthy?

Comment (retroflexer) [reply pkg/operator/etcdmemberscontroller/etcdmemberscontroller.go]: See the else part (L154-165)

Comment (retroflexer) [reply pkg/operator/etcdmemberscontroller/etcdmemberscontroller.go]: I am not sure I understand the question, Alay. Do you want a different status when minority members are unhealthy?

Comment (alaypatel07) [reply pkg/operator/etcdmemberscontroller/etcdmemberscontroller.go]: no, I missed the `else if` was just trying to wrap my head around all the cases

**Comment by retroflexer** [comment]
superseded by other changes.
/close

**Pull request closed without merging**